### PR TITLE
Add documentation for Net Core

### PR DESCRIPTION
**Pull request by samhanes**

### CODE REVIEW

**Comment by samhanes** [comment]
I may have broken this with my last PR - I'm getting this error on the `GenerateDocs` build step:
```
Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'FSharp.Compiler.Private, Version=4.4.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
   at Sample.Microsoft.FSharp.Compiler.Interactive.Main.MainMain(String[] argv)
```

Does that make sense to anyone?

**Comment by smoothdeveloper** [comment]
@samhanes  are you using VS 2017? sounds like a case of updating, rebooting and trying again.

File: docs/content/unit-testing.fsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ module MyTests =
 
     //unit test
     let ``best sales rep in Canada``() = 
-        let expected = Some( GetSalesChampion.Record("Jos�", "Saraiva"))
+        let expected = Some( GetSalesChampion.Record("Jos�", "Saraiva"))
```
Comment (smoothdeveloper) [inline docs/content/unit-testing.fsx]: there is some kind of encoding glitch here.

Comment (samhanes) [reply docs/content/unit-testing.fsx]: Fixed - replaced with a unicode é.

**Comment by samhanes** [comment]
@smoothdeveloper I am generally using Code + Ionide, but in the case of this error I am simply running the FAKE script, i.e., `.\build.cmd GenerateDocs`. I have the latest version of VS 2017 installed (15.8.9) and a reboot did not help, unfortunately.

If this is just happening for me I won't worry about it too much - is anyone else able run that build step on this branch?

**Comment by samhanes** [comment]
@smoothdeveloper Are you able to run the GenerateDocs target on this branch?

**Comment by vasily-kirichenko** [comment]
Thanks! Published https://fsprojects.github.io/FSharp.Data.SqlClient/netcore.html

**Pull request merged**